### DISCUSSION TITLE
R68S R22.7.16版软件源刷不出ipk

### DISCUSSION THREAD

**Discussion by xaqingfeng** [coolsnowwolf/lede#9787] [R68S / 电犀牛专区] [answered]
系统-软件包，发行版软件源刷不出来

**Comment by p-w-p** [comment]
imm源的buyvm没开机，现在应该可以了

**Comment by gd646217009** [comment, marked as answer]
我也遇到同样的问题，目前的解决方案是不用immortalwrt的源了， 换回lean大的源（可以正常更新），外挂一个自定义软件源，插件全而且更新稳定。

**Reply by xaqingfeng** [reply]
能不能发一下lean大的源？找不到passwall和hello word

**Reply by xaqingfeng** [reply]
现在发行版软件源可以刷出来了，可安装不了passwall和vssr，反馈是与配置的架构不兼容

**Reply by p-w-p** [reply]
> 现在发行版软件源可以刷出来了，可安装不了passwall和vssr，反馈是与配置的架构不兼容

同有这个需求QWQ
passwall是缺kmod-ipt-iprange和kmod-ipt-socket
vssr是缺libmbedtls12，手动装上之后
 * check_data_file_clashes: Package luci-lib-ipkg wants to install file /usr/lib/lua/luci/model/ipkg.lua
被占用了

**Reply by xaqingfeng** [reply]
说明这个版本不完善，还存在很多bug

**Reply by p-w-p** [reply]
> 说明这个版本不完善，还存在很多bug

看了下，lean发7.19了，说是解决了passwall的依赖问题
https://github.com/coolsnowwolf/lede/releases/download/20220716/openwrt-rockchip-R22.7.19-docker-fastrhino-r68s-squashfs-sysupgrade.rar

**Reply by xaqingfeng** [reply]
搞了个乌龙，7.19是R5S固件，怎么放到R68S这里了，刷机后根本不能用，识别不出来。

**Reply by p-w-p** [reply]
> 搞了个乌龙，7.19是R5S固件，怎么放到R68S这里了，刷机后根本不能用，识别不出来。

昨晚是发了r68s和h68k的，撤包了XD
忘记改回复了，sry
